### ISSUE TITLE
Versions confusion

### ISSUE THREAD

**Issue by xeruf** [Selekkt/skelet#36]
The landing page says v3, but the repo says v5, are there any differences to be aware of?
https://selekkt.dk/skelet/v3/

**Comment by thagxt** [comment]
Yes, so the URL was not changed because the info/docs you see on that page mostly relate to v3. I didn't have time to rewrite the docs, so some stuff is most likely outdated and new features aren't available there as well as some old features that were removed in new versions such as v5. 

I'll do my best to release [v6](https://github.com/Selekkt/skelet/commits/next/css/skelet.css#:Rat5rab:) by end of December... so you might want to look into that instead of v5.
